profile_collection/startup/10-motors.py

```diff
@@ -18,3 +20,12 @@
 
 
 shctl1 = EpicsMotor('XF:28IDC-ES:1{Sh2:Exp-Ax:5}Mtr', name='shctl1')
+
+
+class FilterBank(ophyd.Device):
+    flt1 = ophyd.Component(EpicsSignal, '1-Cmd', string=True)
+    flt2 = ophyd.Component(EpicsSignal, '2-Cmd', string=True)
+    flt3 = ophyd.Component(EpicsSignal, '3-Cmd', string=True)
+    flt4 = ophyd.Component(EpicsSignal, '4-Cmd', string=True)
+
+pd = EpicsSignal('XF:28IDC-BI:1{IM:02}Pos-Cmd', name='pd', string=True)
```

Comment:
Might want to rename this to avoid name collision with the common alias for `pandas`.